python/lsst/ts/m2gui/utils.py

```diff
@@ -222,3 +224,58 @@ def get_num_actuator_ring(ring):
         return 18
     else:
         raise ValueError(f"Not supported ring: {ring!r}")
+
+
+def prompt_dialog_warning(description, is_prompted=True):
+    """Prompt the warning dialog.
+
+    The user must react to this dialog. The rest of the GUI is blocked until
+    the dialog is dismissed.
+
+    Parameters
+    ----------
+    description : `str`
+        Description that will be shown in the dialog.
+    is_prompted : `bool`, optional
+        Dialog will be prompted or not. The value of False is used in the unit
+        test only. (the default is True)
+    """
+
+    dialog = QMessageBox()
+    dialog.setIcon(QMessageBox.Warning)
+    dialog.setText(description)
+
+    if is_prompted:
+        dialog.exec()
+
+
+def run_command(command, *args, is_prompted=True):
```

Comment:
Add **kwargs as well. You might soon hit commands that require that _(it also demonstrates advantages of the decorator approach when you don't need to care about that, but your choice..)_